docs/_posts/2015-12-09-ismounted-antipattern.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "isMounted is an Antipattern"
+author: jimfb
+---
+
+The primary use case for `isMounted()` is to avoid calling `setState()` after a component has unmounted, because calling `setState()` after a component has unmounted will emit a warning. The “setState warning” exists to help you catch bugs, because calling `setState()` on an unmounted component is an indication that your app/component has somehow failed to clean up properly. Specifically, calling `setState()` in an unmounted component means that your app is still holding a reference to the component after the component has been unmounted - which often indicates a memory leak!
+
+To avoid the error message, people often add lines like this:
+
+```js
+if(this.isMounted()) this.setState({...}); // This is bad.
```

Comment:
Yeah, I kind of liked that it was one line, but I can reformat.  I think I want to keep the "this is bad" comment.  Normally I'd use the red antipattern code boxes that we have internally, but I don't think we have those yet on our public documentation.  I want to make it abundantly clear at a glance that the code should not be copy-pasted into an application.